packaged_releases/bundled/README.md

```diff
@@ -58,3 +58,75 @@ Get the SHA256 checksum (if needed):
 ```
 shasum -a 256 azure-cli_bundle_{VERSION}.tar.gz
 ```
+
+Using the Bundled Installer
+===========================
+Where does this install fit in with everything else?
+----------------------------------------------------
+- It behaves similarly to the interactive script except it’s reproducible, scriptable (non-interactive), configurable and doesn’t rely on PyPI (can be run offline).
+- It’s ideal as part of a CI/production build system.
+- It makes it simpler to install on other Linux distros (e.g. RHEL).
+- More info at #3376
+ 
+Prerequisites
+-------------
+- Linux or macOS
+- Python 2.7+ or Python 3.5+
```

Comment:
Should we add a line here `for windows, there is an installer for similar purpose`